new-dti-website/components/team/MemberDisplay.tsx

```diff
@@ -109,21 +115,17 @@ const MemberDisplay: React.FC = () => {
                 className="grid lg:grid-cols-4 md:grid-cols-3 xs:grid-cols-2 md:gap-10 
                 xs:gap-x-1.5 xs:gap-y-5 md:mt-10 xs:mt-5"
               >
-                {alumniMembers.members.map((member, index) =>
-                  index <= 5 ? (
-                    <a href={member.linkedin ?? undefined} key={index}>
-                      <MemberCard
-                        {...member}
-                        roleDescription={member.roleDescription as RoleDescription}
-                        cardState={undefined}
-                        image={`team/${member.netid}.jpg`}
-                        key={member.netid}
-                      />
-                    </a>
-                  ) : (
-                    <></>
-                  )
-                )}
+                {orderedAlumni.map((member, index) => (
+                  <a href={member.linkedin ?? undefined} key={index}>
```

Comment:
I think it would be better to use a 'next/Link' here if we're keeping it consistent! Also I can't remember if there was a reason we didn't want the link to create a new tab? If we did then We could wrap the a tag within a Link as well like: 

`<Link href={member.linkedin ?? '#'}>
  <a target="_blank" rel="noopener noreferrer">
    <MemberCard
      {...member}
      roleDescription={member.roleDescription as RoleDescription}
      cardState={undefined}
      image={`team/${member.netid}.jpg`}
    />
  </a>
</Link>`

Again wondering if @Bookie0 has thoughts on this!

Reply:
I didn't change this one because I couldn't find a way to "disable" the link behavior for the case that `member.linkedin` is `null` or `undefined`. Using `#` I believe just tries to construct a link `https://new.cornelldti.org/team#`.

So I _think_ (please correct me if I'm wrong) that I'd be forced to write something like this? Is there a better way?
```
                  member.linkedin ? (
                    <Link
                      href={member.linkedin}
                      key={index}
                      target="_blank"
                      rel="noopener noreferrer"
                    >
                      <MemberCard
                        {...member}
                        roleDescription={member.roleDescription as RoleDescription}
                        cardState={undefined}
                        image={`team/${member.netid}.jpg`}
                        key={member.netid}
                      />
                    </Link>
                  ) : (
                    <MemberCard
                      {...member}
                      roleDescription={member.roleDescription as RoleDescription}
                      cardState={undefined}
                      image={`team/${member.netid}.jpg`}
                      key={member.netid}
                    />
                  )
```

Reply:
I'm on board with opening up a new page in a new tab though!

Reply:
You're absolutely right — checking for null or undefined before rendering the link would be necessary! 
Thank you for catching this! You can disregard this comment and sorry for the confusion!